pyracing/response_objects/iracing_data.py

```diff
@@ -28,3 +28,48 @@ def value_or_none(dict, field_name):
             return dict[field_name]
         else:
             return None
+
+
+class CarClass:
+    def __init__(self, dict):
+        self.rel_speed = dict['relspeed']  # Speed ranking to other classes
+        self.lower_name = dict['lowername']
+        self.custid = dict['custid']
+        self.class_full_name = dict['name']
+
+        # Inject into subclass CarsInClass?
+        self.carsinclass = dict['carsinclass']
+        self.class_id = dict['id']
+        self.class_shortname = dict['shortname']
+
+    class CarsInClass:
```

Comment:
I think what we want here is a separate model just called `Car` and the `carsinclass` attribute would store a list of cars. That way we have a model we can use for any time we get a car, and it doesn't have to be nested inside of something else